std_msgs/Header header
float32 actual_pose_x
float32 actual_pose_y
float32 actual_pose_theta
float32 target_pose_x
float32 target_pose_y
float32 target_pose_theta
float32 leader_pose_x
float32 leader_pose_y
float32 leader_pose_theta
float32 error_x 
float32 error_y
float32 error_theta
float32 target_velocity_x
float32 target_velocity_y
int16   robot_id